packages/core/src/render3/index.ts

```diff
@@ -128,6 +128,8 @@ export {
   ɵɵtextInterpolate7,
   ɵɵtextInterpolate8,
   ɵɵtextInterpolateV,
+
+  ɵsetShouldThrowErrorOnUnknownElement
```

Comment:
May be we can call is strict mode?
```suggestion
  ɵsetUnknownElementStrictMode
```